tests/end_to_end/test_docker_monitoring/data/test_cases/cases_test_docker_monitoring.yaml

```diff
@@ -0,0 +1,48 @@
+---
+- name: docker_pull
+  description: Test if an alert is generated when using the `pull` command.
+  configuration_parameters: null
+  metadata:
```

Comment:
Add `rule.level`, please

Reply:
Done in https://github.com/wazuh/wazuh-qa/commit/ced7d8990b0043155fe47e9532b36bae152b0bc0